scripts/toltec/version.py

```diff
@@ -145,21 +145,13 @@ def __init__(
     def parse(dependency: str) -> "Dependency":
         """Parse a dependency specification."""
         original = dependency
-        colon = dependency.find(":")
+        kind = DependencyKind.Host
 
-        if colon == -1:
-            kind = DependencyKind.Host
-        else:
-            for enum_kind in DependencyKind:
-                if enum_kind.value == dependency[:colon]:
-                    kind = enum_kind
-                    dependency = dependency[colon + 1 :]
-                    break
-            else:
-                raise InvalidDependencyError(
-                    f"Unknown dependency type \
-'{dependency[:colon]}'"
-                )
+        for enum_kind in DependencyKind:
```

Comment:
No more raising errors?

Reply:
Since there is an ambiguity between dependency types specifiers and epochs, I first check whether any known dependency type is present and otherwise let it fall through as an epoch. If some invalid dependency type is given, it’ll be processed as an epoch, which will also raise an error since epochs need to be numeric.

Reply:
Alright, that's good enough for me. You'll need to re-approve the changes.

Reply:
Great! I still need to test that the install process goes smoothly before approving. I’ll do this later in the day.